pkg/cmd/browse/browse.go

```diff
@@ -252,6 +260,13 @@ func isNumber(arg string) bool {
 	return err == nil
 }
 
+// sha1 and sha256 are supported
+var commitHash = regexp.MustCompile(`\Acommit/[a-f0-9]{7,64}\z`)
```

Comment:
What @samcoe was proposing is that we drop the `commit/` prefix from the argument and just assume that any argument that matches `[a-f0-9]{7,64}` is a git commit SHA. Do you think you could amend the PR with that logic?

Reply:
fixed. Could you check it again?